docs/userguide/entry_point.rst

```diff
@@ -94,9 +113,88 @@ where ``name`` is the name for the script you want to create, the left hand
 side of ``:`` is the module that contains your function and the right hand
 side is the object you want to invoke (e.g. a function).
 
+GUI Scripts
+===========
+
 In addition to ``console_scripts``, Setuptools supports ``gui_scripts``, which
 will launch a GUI application without running in a terminal window.
 
+For example, if we have a project with the same directory structure as before,
+with an ``__init__.py`` file containing the following:
+
+.. code-block:: python
+
+    import PySimpleGUI as sg
+
+    def hello_world():
+        sg.Window(title="Hello world", layout=[[]], margins=(100, 50)).read()
+
+Then, we can add a GUI script entry point:
+
+.. tab:: setup.cfg
+
+    .. code-block:: ini
+
+        [options.entry_points]
+        gui_scripts =
+            hello-world = timmins:hello_world
+
+.. tab:: setup.py
+
+    .. code-block:: python
+	
+        from setuptools import setup
+
+        setup(
+            # ...,
+            entry_points={
+                'gui_scripts': [
+                    'hello-world=timmins:hello_world',
+                ]
+            }
+        )
+
+.. tab:: pyproject.toml (**EXPERIMENTAL**) [#experimental]_
+
+   .. code-block:: toml
+
+        [project.gui-scripts]
+        hello-world = "timmins:hello_world"
+
```

Comment:
The note may be obvious, but maybe it is helpful for beginners? Or should we instead just add the dependency requirements directly in the tabs above?

```suggestion

.. note::
   To be able to import `PySimpleGUI`, you need to add `pysimplegui` to your package dependencies.
   See :doc:`/userguide/dependency_management` for more information.
```

Reply:
Yeah, I thought about this too. Your call on this. I think both ways will be alright.

Reply:
Let's add the note, *just-in-case* ™️